doc/classes/EditorSettings.xml

```diff
@@ -812,6 +812,13 @@
 			Input accumulation can be disabled to get slightly more precise/reactive input at the cost of increased CPU usage.
 			[b]Note:[/b] Input accumulation is [i]enabled[/i] by default.
 		</member>
+		<member name="interface/accessibility/accessibility_support" type="int" setter="" getter="">
+			Editor accessibility support mode:
```

Comment:
These kinds of descriptions always make my nose twitch.
```suggestion
			How the editor implements its accessibility support:
```
Or similar. As I've said in the past, I am not too familiar with proper terminology for this.

Reply:
Not sure if `implements` is a proper word here. It's just a toggle enabling/disabling the support.